source/solution_deploy/lib/solution_deploy-stack.ts

```diff
@@ -703,42 +705,43 @@ export class SolutionDeployStack extends cdk.Stack {
-
-          //---------------------------------------------------------------------
-          // Playbook Admin Template Nested Stack
-          //
-          const parmname = file.replace(illegalChars, '');
-          const adminStackOption = new cdk.CfnParameter(this, `LoadAdminStack${parmname}`, {
-            type: 'String',
-            description: `Load CloudWatch Event Rules for ${file}?`,
-            default: 'yes',
-            allowedValues: ['yes', 'no'],
-          });
-          adminStackOption.overrideLogicalId(`Load${parmname}AdminStack`);
-          standardLogicalNames.push(`Load${parmname}AdminStack`);
-
-          const adminStack = new cdk.CfnStack(this, `PlaybookAdminStack${file}`, {
-            templateUrl:
-              'https://' +
-              cdk.Fn.findInMap('SourceCode', 'General', 'S3Bucket') +
-              '-reference.s3.amazonaws.com/' +
-              cdk.Fn.findInMap('SourceCode', 'General', 'KeyPrefix') +
-              '/playbooks/' +
-              template_file,
-          });
-          adminStack.addDependency(stateMachineConstruct);
-          adminStack.addDependency(orchestratorArn);
-
-          adminStack.cfnOptions.condition = new cdk.CfnCondition(this, `load${file}Cond`, {
-            expression: cdk.Fn.conditionEquals(adminStackOption, 'yes'),
-          });
-        }
-      });
+    const items = fs.readdirSync(PB_DIR);
+    items.forEach((file) => {
+      if (!ignore.includes(file)) {
+        const template_file = `${file}Stack.template`;
+
+        //---------------------------------------------------------------------
+        // Playbook Admin Template Nested Stack
+        //
+        const parmname = file.replace(illegalChars, '');
+        const adminStackOption = new cdk.CfnParameter(this, `LoadAdminStack${parmname}`, {
+          type: 'String',
+          description: `Load CloudWatch Event Rules for ${file}?`,
+          default: 'yes',
+          allowedValues: ['yes', 'no'],
+        });
+        adminStackOption.overrideLogicalId(`Load${parmname}AdminStack`);
+        standardLogicalNames.push(`Load${parmname}AdminStack`);
+
+        const adminStack = new cdk.NestedStack(this, `PlaybookAdminStack${file}`);
+        const cfnStack = adminStack.nestedStackResource as cdk.CfnResource;
+        cfnStack.addPropertyOverride(
+          'TemplateURL',
+          'https://' +
+            cdk.Fn.findInMap('SourceCode', 'General', 'S3Bucket') +
+            '-reference.s3.amazonaws.com/' +
+            cdk.Fn.findInMap('SourceCode', 'General', 'KeyPrefix') +
+            '/playbooks/' +
+            template_file
+        );
+        cfnStack.cfnOptions.condition = new cdk.CfnCondition(this, `load${file}Cond`, {
+          expression: cdk.Fn.conditionEquals(adminStackOption, 'yes'),
+        });
+        cfnStack.node.addDependency(stateMachineConstruct);
+        cfnStack.node.addDependency(orchestratorArn);
+
+        this.nestedStacks.push(adminStack as cdk.Stack);
```

Comment:
I'm a little surprised this works just fine. Do they show up in appreg?

Reply:
Yep, this is similar to the approach used in devops monitoring dashboard. I'll DM you the snippet.